tests/testthat.R

```diff
@@ -1,5 +1,12 @@
 library(testthat)
 library(eidith)
 
+
+test_that("ed_tests_report is returning an error", {
+  expect_error(ed_tests_report(), NA)
+}
+)
+
+
 test_check("eidith")
```

Comment:
This test should go in a new file under `tests/testthat` Note the filename structure there, as well as the "if" statements in those files. In tests, first everything in the helpers file is run, then tests are run consecutively.  Things requiring a database to exist have to go after the first set of tests, when data are downloaded, and one should use an if statement to only run the test is there is a database to test against.